tests/integration/routes/test_questionnaire.py

```diff
@@ -44,3 +44,12 @@ def test_head_request_on_block_with_optional_date_answer(self):
         )
         self.head("/questionnaire/date-non-mandatory-block/")
         self.assertStatusOK()
+
+    def test_options_request_exits_before_request_early(self):
```

Comment:
Very minor but the 'early' confused me for a second, not sure it is needed

Reply:
Done